graph/src/firehose/endpoints.rs

```diff
@@ -56,7 +56,9 @@ impl FirehoseEndpoint {
                 .expect("TLS config on this host is invalid"),
             _ => panic!("invalid uri scheme for firehose endpoint"),
         }
-        .connect_timeout(Duration::from_secs(10));
+        .connect_timeout(Duration::from_secs(10))
+        .http2_keep_alive_interval(Duration::from_secs(30))
+        .http2_adaptive_window(true);
```

Comment:
This causes disconnections when connecting to firehose through a GCP Load Balancer. (`error reading a body from connection: unexpected end of file`)

Reply:
Thanks for testing that! I've removed this, and took a different approach of setting a very large window size.